qiskit_nature/runtime/vqe_client.py

```diff
@@ -30,6 +30,10 @@
 from qiskit.opflow import OperatorBase, PauliSumOp
 from qiskit.quantum_info import SparsePauliOp
 
+# Introduced new type to maintain readability.
+_T = TypeVar("_T")  # Pylint does not allow single character class names.
+ListOrDict = Union[List[Optional[_T]], Dict[str, _T]]
```

Comment:
Rather than writing your own type variable you can simply use
```python
from qiskit_nature import ListOrDictType
```

Reply:
Thanks for the comment. I just blatantly copied it from minimum_eigen_solver.py in qiskit algorithms; of course I'll change it to the nicer import.

Reply:
In the dev branch of Terra the ListOrDict type has been split out so it can be directly imported elsewhere. However at present we cannot use that here since we need to stay compatible with 0.20 release. For me having a copy here so it matches what is in MinimumEigenSolver in the interim seems ok. It can be replaced with with the import at some later point.

Reply:
Do I understand it correctly that `from qiskit_nature import ListOrDictType` is not compatible with the 0.20 Terra release? So, I should change it back to the copied lines from `minimum_eigen_solver.py`?

Reply:
ListOrDictType is the same thing just with a different name https://github.com/Qiskit/qiskit-nature/blob/8367463a47fcb46770b310d2c5b2bf7a067398bf/qiskit_nature/__init__.py#L62-L64 It makes the interface look as though the type might be different, due to importing it as a different name, but in the end as you can see its the same thing.

Reply:
Thanks for the explanation. Somewhat confusing with the different names `ListOrDict` and `ListOrDictType` but now I get it.